### DISCUSSION TITLE
Technical Design Discussion

### DISCUSSION THREAD

**Discussion by sushil-relicx** [getgauge/taiko#1733] [General]
We are a DevOps startup and are considering building our product using Taiko. We would like to have a call with the Taiko technical lead to get their perspective on our design. Can you please suggest best way to do this? 

Thanks.

**Comment by zabil** [comment]
That's great!

Is the product open source? As Taiko is open source it's best to have these discussions here on Github so that it can be referred by others.

We also recommend checking https://github.com/getgauge/taiko/wiki for documentation about the architecture.

**Comment by sushil-relicx** [comment]
We are a software-as-a-service offering but we plan to contribute significantly to the open source community, including Taiko. We will definitely post our queries to this forum but the project that we are working on involved automatically generating Taiko scripts. Meaning those scripts won't be written by a human user which seems to be the design point so far. So, we wanted to have a design discussion with the technical team to see how best we can contribute towards enhancing Taiko to make automated test generation possible.

**Comment by zabil** [comment]
The team is planning to set up and open weekly tech calls starting next week. The details will be up soon on the https://docs.taiko.dev/contributing/ page.

**Reply by sushil-relicx** [reply]
Thank you. We are looking forward to joining those calls.

**Reply by zabil** [reply]
Details are up at https://github.com/getgauge/taiko/discussions/1741
